aplt/scenarios.py

```diff
@@ -0,0 +1,26 @@
+"""Several basic push scenarios"""
+from twisted.python import log
+
+from aplt.commands import (
+    connect,
+    hello,
+    register,
+    send_notification,
+    expect_notification,
+    unregister,
+    disconnect,
+    ack,
+    random_channel_id,
+)
+
+
+def basic():
```

Comment:
TODO: probably want to document this a bit better (e.g. why "yield" prefix for calls.) Best to presume zero prior knowledge if handing this over to a different team.